builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -139,40 +145,282 @@ class IntValidator implements TypeValidator {
   }
 }
 
+class StringValidator implements TypeValidator {
+  public void isValid(def aValue) {
+    if(!aValue instanceof String)
+    throw new IllegalArgumentException(aValue);
```

Comment:
If we are throwing the same exception, can we at least and a more informative message to say what the issue is and what type we are dealing with. Something like for all Validators:

> throw new IllegalArgumentException("Invalid String: " + aValue + " is of type: " + aValue?.class);

Reply:
Added the proper error messages